api/src/pcapi/routes/adage_iframe/serialization/adage_authentication.py

```diff
@@ -38,6 +39,7 @@ class AuthenticatedResponse(BaseModel):
     lat: float | None = None
     lon: float | None = None
     favoritesCount: int = 0
+    offerCount: int = 0
```

Comment:
[détail / nommage] Puisque "favorite**s**Count" a un "s", "offer**s**Count" pourrait aussi en avoir un, par ~solidarité~ homogénéité ?